src/css/components/_captions-settings.scss

```diff
@@ -30,7 +60,7 @@
 
 .vjs-text-track-settings fieldset span {
   display: inline-block;
-  margin-left: 5px;
+  margin-left: 0px;
```

Comment:
I think this line is unnecessary now.

Reply:
yes you are correct, just pushed to remove this.